FordLabs/peoplemover
# Refactor services to be objects vs classes

## Issue
Resolves N/A

## What was done
- [x] Refactor services to be objects vs classes

## How to test
1. Confirm nothing functionally has changed in the app and all tests pass
2. Confirm no functionality in services has changed
